app/src/main/java/org/lineageos/glimpse/thumbnail/MediaViewerAdapter.kt

```diff
@@ -40,7 +41,10 @@ class MediaViewerAdapter(
     )
 
     override fun onBindViewHolder(holder: MediaViewHolder, position: Int) {
-        getMediaFromMediaStore(position)?.let { holder.bind(it, position) }
+        getMediaFromMediaStore(position)?.let {
+            holder.bind(it, position)
+            mediaUpdateCallback(it, position)
```

Comment:
Why is this called here? onBindViewHolder is called before the view is shown

Reply:
Ah you check the position before doing stuff

Reply:
Right, on viewAttached is better

Reply:
NAK, breaks callback expectations